scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -634,16 +634,7 @@ <h4 class="modal-title">Cancel Sanger sequencing</h4>
             <a href="{{ gene.oncokb_link }}" class="btn btn-outline-secondary" target="_blank">OncoKB</a>
             <a href="{{ gene.civic_link }}" class="btn btn-outline-secondary" target="_blank">CIViC</a>
           {% endif %}
-          <form action="http://v1.marrvel.org/search" method="POST" target="_blank" style="display:inline;">
-            <button type="submit" class="btn btn-outline-secondary">MARRVEL</button>
-            <input type="hidden" name="variantGeneId" value="{{ variant.chromosome }}:{{ variant.position }} {{ variant.reference }}>{{ variant.alternative }}">
-            <input type="hidden" name="inGeneSymbol" value="{{ gene.common.hgnc_symbol if gene.common }}">
-            <input type="hidden" name="OMIMdbCheck" value="on">
-            <input type="hidden" name="ExACdbCheck" value="on">
-            <input type="hidden" name="Geno2MPdbCheck" value="on">
-            <input type="hidden" name="DGVdbCheck" value="on">
-            <input type="hidden" name="DECIPHERdbCheck" value="on">
-          </form>
+            <a href="http://marrvel.org/human/variant/{{ variant.chromosome }}:{{ variant.position }} {{ variant.reference }}>{{variant.alternative}}" class="btn btn-outline-secondary" target="_blank">MARRVEL</a>
```

Comment:
Tested and it works:

![image](https://user-images.githubusercontent.com/28093618/101343260-083d3680-3884-11eb-8fa4-9d8cbeabdaea.png)